### PR TITLE
BK-787 Create bulk items

### PR DESCRIPTION
**Pull request by GLagunilla**
BK-787

### CODE REVIEW

File: src/Management/ManagementClient.php [hunk 1]
```diff
@@ -615,4 +615,58 @@ public function deleteItemsInBulkInTemporalIndex($hashId, $itemId, $params)
...
+    /**
+     * Given a hashId, index id and items data, updates items
+     *
+     * @param string $hashId
+     * @param string $itemId
+     * @param array $params
+     * @return HttpResponseInterface
+     * @throws ApiException
+     */
+    public function updateItemsInBulk($hashId, $itemId, $params)
```
Comment (agutierrezrodriguez) [inline src/Management/ManagementClient.php:646]: same here

File: src/Management/ManagementClient.php [hunk 2]
```diff
@@ -615,4 +615,58 @@ public function deleteItemsInBulkInTemporalIndex($hashId, $itemId, $params)
...
+    /**
+     * Given a hashId, index id and items id, deletes items
+     *
+     * @param string $hashId
+     * @param string $itemId
+     * @param array $params
+     * @return HttpResponseInterface
+     * @throws ApiException
+     */
+    public function deleteItemsInBulk($hashId, $itemId, $params)
```
Comment (agutierrezrodriguez) [inline src/Management/ManagementClient.php:664]: and same here

**Review by agutierrezrodriguez — CHANGES_REQUESTED** [review]
I think that you want to write "index id" or "index name" or something related.

**Comment by GLagunilla** [comment]
Yes, it should be $indexName :man_facepalming:

**Review by agutierrezrodriguez — APPROVED** [review]

**Review by vpoten — APPROVED** [review]

**Pull request merged**